coordinator/src/dlc_handler.rs

```diff
@@ -108,7 +121,9 @@ impl DlcHandler {
                     is_offer: false, ..
                 },
             ..
-        }) = signed_dlc_channels.iter().find(|c| c.counter_party == peer)
+        }) = signed_dlc_channels
+            .iter()
+            .find(|c| c.counter_party == to_secp_pk_29(peer))
```

Comment:
❓ I guess this is now needed because we are depending on two different bitcoin versions, right? Should we maybe leave a TODO here to change that back once, `rust-dlc` has also been updated?

Reply:
I don't think we need to leave TODOs because the compiler will just complain once the types don't match.